css-positioning/css-positioning/absolute.css

```diff
@@ -0,0 +1,46 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  left: 220px;
```

Comment:
Woah, that's a big movement!! You want this in the top right, yes? Moving it form the left side seems like a lot of work to get it to the right 🤔